lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/pt/PtRouting.java

```diff
@@ -0,0 +1,183 @@
+
+import com.google.common.collect.Iterables;
+import com.graphhopper.GHResponse;
+import com.graphhopper.GraphHopperConfig;
+import com.graphhopper.ResponsePath;
+import com.graphhopper.Trip;
+import com.graphhopper.config.Profile;
+import com.graphhopper.gtfs.GraphHopperGtfs;
+import com.graphhopper.gtfs.PtRouter;
+import com.graphhopper.gtfs.PtRouterImpl;
+import com.graphhopper.gtfs.Request;
+import com.graphhopper.util.StopWatch;
+import com.graphhopper.util.TranslationMap;
+import org.apache.commons.lang3.Validate;
+import org.locationtech.jts.geom.Coordinate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class PtRouting {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PtRouting.class);
+
+    private final ExecutorService routingExecution = Executors.newSingleThreadExecutor();
+
+    private PtRouter ptRouter;
+    private GraphHopperGtfs graphHopperGtfs;
+
+    private LocalDateTime scheduleDateTime;
+    private ZoneId timeZone;
+
+    public void initialize(CPublicTransportRouting routingConfiguration, File configurationLocation) {
+        if (!routingConfiguration.enabled) {
+            return;
+        }
+
+        Path baseDirectory = configurationLocation.toPath();
+
+        GraphHopperConfig ghConfig = new GraphHopperConfig()
+                .putObject("import.osm.ignored_highways", "motorway,trunk,primary")
+                .putObject("graph.location", baseDirectory.resolve("ptgraph").toAbsolutePath().toString())
+                .putObject("datareader.file", baseDirectory.resolve(routingConfiguration.osmFile).toAbsolutePath().toString())
+                .putObject("gtfs.file", baseDirectory.resolve(routingConfiguration.gtfsFile).toAbsolutePath().toString())
+                .setProfiles(Collections.singletonList(new Profile("foot").setVehicle("foot")));
+
+        scheduleDateTime = LocalDateTime.parse(routingConfiguration.scheduleDateTime, DateTimeFormatter.ISO_DATE_TIME);
+        timeZone = ZoneId.of(ZoneId.SHORT_IDS.get(routingConfiguration.timeZone));
+
+        final StopWatch sw = new StopWatch();
+        sw.start();
+        graphHopperGtfs = new GraphHopperGtfs(ghConfig);
+        graphHopperGtfs.init(ghConfig);
+        graphHopperGtfs.importOrLoad();
+        sw.stop();
+        LOG.debug("Took {} ms to load public transport router.", sw.getMillis());
+
+        LOG.info("setting ptRouter");
```

Comment:
```suggestion
        LOG.info("Initializing Public Transport Router");
```
or put the log-statement after the initialization and do
```suggestion
        LOG.info("Initialized Public Transport Router");
```